web/client/epics/__tests__/maplayout-test.js

```diff
@@ -14,8 +14,12 @@ const {closeIdentify, purgeMapInfoResults, noQueryableLayers} = require('../../a
 
 const {updateMapLayoutEpic} = require('../maplayout');
 const {testEpic, addTimeoutEpic, TEST_TIMEOUT} = require('./epicTestUtils');
+const ConfigUtils = require('../../utils/ConfigUtils');
 
 describe('map layout epics', () => {
+    beforeEach(() => {
```

Comment:
i think afterEach should be more correct, or it will work only if the test is not the last
```suggestion
    afterEach(() => {
```